tap_kustomer/streams.py

```diff
@@ -56,6 +56,124 @@ def get_child_context(
             "id": record["id"],
         }
 
+    def post_process(
+        self,
+        row: dict,
+        context: dict | None = None,  # noqa: ARG002
+    ) -> dict | None:
+        if isinstance(
+            row["attributes"].get("firstMessageIn", {}).get("meta", {}).get("to"),
```

Comment:
@prakharcode Can you refactor this so the nested field is looked up only once.

e.g.

```python
def post_process(
    self,
    row: dict,
    context: dict | None = None,  # noqa: ARG002
) -> dict | None:
    first_message_to = row["attributes"].get("firstMessageIn", {}).get("meta", {}).get("to")
   first_message_cc = ...
    last_message_to = ...
    if isinstance(message_to, dict):
        ...
```